exercises/bob/.meta/generator/test_template.erb

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env ruby
+require 'minitest/autorun'
+require_relative '<%= exercise_name %>'
+
+# Common test data version: <%= canonical_data_version %> <%= abbreviated_commit_hash %>
+class <%= exercise_name_camel %>Test < Minitest::Test
+  def bob
+    Bob
+  end
+
+  def feedback(text)
+    "Bob hears #{text.inspect}, and.."
+  end
```

Comment:
This really belongs in the generator, the feedback should be expanded before it gets added to the test.